dhis-2/dhis-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/view/PageTest.java

```diff
@@ -122,6 +124,21 @@ void shouldSetNextPage() {
         "fields=displayName");
   }
 
+  @Test
+  void shouldFailWhenPageIsNotNullAndSmallerThanOne() {
```

Comment:
these tests do not belong in the `view/PageTest` as you are testing `tracker/PageParams`